### PR TITLE
Updated test scenarios to work on containers

### PR DESCRIPTION
**Pull request by matejak**
Added package metadata, and made scenarios more robust - not all files may really be there all the time.

### CODE REVIEW

File: linux_os/guide/system/logging/log_rotation/ensure_logrotate_activated/tests/logrotate_conf_extra_monthly.fail.sh [hunk 1]
```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
 
+# packages: logrotate,crontabs
```
Comment (ggbecker) [inline linux_os/guide/system/logging/log_rotation/ensure_logrotate_activated/tests/logrotate_conf_extra_monthly.fail.sh:3]: I don't think `colon` is the right syntax here:

https://github.com/ComplianceAsCode/content/blob/22ef2f61b24ff4686d745ba68656ca3b88a99e27/tests/ssg_test_suite/rule.py#L311

Comment (ggbecker) [reply linux_os/guide/system/logging/log_rotation/ensure_logrotate_activated/tests/logrotate_conf_extra_monthly.fail.sh:3]: @matejak this should be the only change required here. Can you update the PR?

Comment (matejak) [reply linux_os/guide/system/logging/log_rotation/ensure_logrotate_activated/tests/logrotate_conf_extra_monthly.fail.sh:3]: Correct, thanks!

**Comment by ggbecker** [comment]
I've tested in a few environments and they all look good. If they contain changes from https://github.com/ComplianceAsCode/content/pull/6710 which is already in main branch.

LGTM

**Pull request merged**